pyclam/manifold.py

```diff
@@ -405,6 +398,9 @@ def __hash__(self):
     def __contains__(self, cluster: 'Cluster') -> bool:
         return cluster in self.clusters.keys()
 
+    def distance(self, *, x1: Union[int, List[int], None], x2: Union[int, List[int], None], p1: Data = None, p2: Data = None) -> np.ndarray:
+        return self.manifold.distance(x1=x1, x2=x2, p1=p1, p2=p2)
```

Comment:
Disagree with this implementation. I'd rather see just x1 x2. p1 p2 is super weird here for me

Reply:
https://www.python.org/dev/peps/pep-0570/